fhir-persistence-schema/src/main/resources/add_resource_template.sql

```diff
@@ -58,11 +58,6 @@ BEGIN
   SELECT t.tenant_id INTO {{SCHEMA_NAME}}.session_tenant
     FROM {{SCHEMA_NAME}}.tenants t
    WHERE t.tenant_name = p_tenant_data_id;
-  
-  -- Stop right here if we don't have a valid tenant
-  IF ({{SCHEMA_NAME}}.session_tenant IS NULL) THEN
-  	SIGNAL SQLSTATE '99401' SET MESSAGE_TEXT = 'NOT AUTHORIZED: INVALID TENANT ID';
-  END IF;
```

Comment:
I'm wondering, is this store procedure really used?

Reply:
I think that was the original and replaced by add_any_resource

2020-04-29 15:05:36.966 00000001    INFO   com.ibm.fhir.schema.app.Main Opening connection to: jdbc:db2://localhost:50000/FHIRDB
2020-04-29 15:05:37.585 00000001    INFO .database.utils.db2.Db2Adapter Create or replace procedure FHIR_ADMIN.SET_TENANT
2020-04-29 15:05:37.648 00000001    INFO .database.utils.db2.Db2Adapter Create or replace procedure FHIRDATA.ADD_CODE_SYSTEM
2020-04-29 15:05:37.722 00000001    INFO .database.utils.db2.Db2Adapter Create or replace procedure FHIRDATA.ADD_PARAMETER_NAME
2020-04-29 15:05:37.752 00000001    INFO .database.utils.db2.Db2Adapter Create or replace procedure FHIRDATA.ADD_RESOURCE_TYPE
2020-04-29 15:05:37.781 00000001    INFO .database.utils.db2.Db2Adapter Create or replace procedure FHIRDATA.ADD_ANY_RESOURCE

Reply:
I'm going to unresolve, confirm and remove